tools/github-actions/audit/packaged-action/index.js

```diff
@@ -26189,9 +26189,9 @@ var __importStar = (this && this.__importStar) || function (mod) {
     return result;
 };
```

Comment:
It seems strange only generated files have been modified in this PR and not the original main.ts?

Reply:
She modified the nx.json, so it invalidated the cache, regenerated the code and linted it

Reply:
why are we even linting the generated js code ?

Reply:
still, I don't think it is good that we have changes on something unrelated and that does not match the initial ts file

Reply:
Also, we should not lint generated files